src/pages/admin/SeminarManage/Modal/AddSeminarModal.tsx

```diff
@@ -9,31 +10,39 @@ interface AddSeminarModalProps {
 }
 
 const AddSeminarModal = ({ open, setOpen }: AddSeminarModalProps) => {
-  const [date, setDate] = useState<DateTime | null>(null);
+  const [seminarDate, setSeminarDate] = useState(DateTime.now());
+
+  const { mutate: addSeminar, isSuccess: isAddSeminarSuccess } = useAddSeminarMutation();
+
   const handleClose = () => {
     setOpen(false);
-    setDate(null);
+    setSeminarDate(DateTime.now());
   };
 
   const handleAddSeminarButtonClick = () => {
-    // TODO
-    setDate(null);
+    addSeminar(seminarDate);
```

Comment:
```suggestion
    addSeminar(seminarDate, {
      onSuccess: () => {
        handleClose();
      },
    });
```

https://github.com/KEEPER31337/Homepage-Front-R2/pull/695/files#r1332902919 이렇게 처리하면 호출 성공 시 바로 동기화된 처리가 가능할 것 같습니다! (delete 쪽에도 동일하게 처리하면 될 것 같네요!)

Reply:
우엉.. 전 바봅니다...